### DISCUSSION TITLE
ORIN AGX WANTED TO EXPAND THE LIBRARY

### DISCUSSION THREAD

**Discussion by Rubberazer** [Rubberazer/JETGPIO#21] [Announcements]
To any companies out there:

I am looking to expand the library a little more and give support to the Orin AGX, which is the only Orin not supported at the moment. In order to  do that properly I need to put my hands into one of those and so:

- If you have used this library with your Nano or Orin Nano/NX and you are happy with the outcome (and potentially made some customer happy too)

- If you have a spare Orin AGX dev kit or similar in the office/warehouse and want this library to support that model, send one over and I will get the last Orin model supported in a matter of days.

If you are interested contact me here: rubberazer@outlook.com

**Comment by Hubuwenxin** [comment]
It's really suitable if can be used on agx orin

**Reply by Rubberazer** [reply]
Well yes, that's the idea. Let's see if somebody sends me one over and I will add support for the AGX, I could do it now, but it would be impossible to test it without the hardware and I don't like the idea of releasing something completely untested to the public.

**Reply by Rubberazer** [reply]
Please define "parallel communication" I dont know what you meant with that. Also, please if any issue or question just start another discussion thread, this one is about people who want to help expand the library.

**Reply by Hubuwenxin** [reply]
sorry， i already deleted and published the problem at the discuss part.
Looking forward to your reply!
